unittest/CMakeLists.txt

```diff
@@ -118,9 +126,11 @@ ADD_PINOCCHIO_UNIT_TEST(liegroups)
 ADD_PINOCCHIO_UNIT_TEST(regressor)
 ADD_PINOCCHIO_UNIT_TEST(version)
 ADD_PINOCCHIO_UNIT_TEST(copy)
-MAKE_DIRECTORY("${CMAKE_CURRENT_BINARY_DIR}/serialization-data")
-ADD_PINOCCHIO_UNIT_TEST(serialization)
-ADD_TEST_CFLAGS(serialization '-DTEST_SERIALIZATION_FOLDER=\\\"${CMAKE_CURRENT_BINARY_DIR}/serialization-data\\\"')
+if(NOT WIN32)
```

Comment:
@francois-keith Do you know why this test is not passing?

Reply:
Yes, I just figured it out. I used to have a link issue, but it was due to a faulty boost installation. Once corrected, I had an input stream error in one of the unit tests, solved by 02e4c3d